GetAssetTrigger/index.ts

```diff
@@ -0,0 +1,74 @@
+import { AzureFunction, Context, HttpRequest } from "@azure/functions"
+import axios, { AxiosError } from "axios"
+import type { Asset } from "@equinor/data-marketplace-models"
+import { config } from "../config"
+
+const GetAssetTrigger: AzureFunction = async function (context: Context, req: HttpRequest): Promise<void> {
+  const { id } = context.bindingData
+
+  try {
+    const { data: collibraAsset } = await axios.get(`${config.COLLIBRA_BASE_URL}/assets/${id}`, {
+      headers: {
+        authorization: req.headers.authorization,
+      },
+    })
+
+    const asset: Asset = {
+      createdAt: new Date(collibraAsset.createdOn),
+      description: "",
+      excerpt: "",
+      id: collibraAsset.id,
+      provider: {
+        id: "",
+        name: "Collibra",
+      },
+      qualityScore: 0,
+      rating: 0,
+      tags: [],
+      type: {
+        id: collibraAsset.type.id,
+        name: collibraAsset.type.name,
+      },
+      updatedAt: new Date(collibraAsset.lastModifiedOn),
+      updateFrequency: "",
+    }
```

Comment:
I have to admit that I don't remember all the asset details here, but I guess this is more or less what we discussed during the workshop 😅